with rank api, some problem while testing. please check

vinay36999 opened the pull request.

Commit:
with rank api, some problem while testing. please check

lwcf1122 commented:
@vinay36999 code looks good to me. Have you tested locally? Did testing pass?

lwcf1122 commented:
@vinay36999  please also update README.md with the sample request and response for Rank resource APIs.

Commit:
Changes to be committed:
	modified:   src/main/java/com/recipe/rest/entity/RecipeDO.java
	modified:   src/main/java/com/recipe/rest/resource/RankResource.java
	modified:   src/main/resources/persistence.xml
In persistence.xml class, when I add RankDO and do mvn jetty:run I am
getting error. Can't figure out why!

Commit:
updated README.md
